sdk/src/main/scala/io/horizen/params/MainNetParams.scala

```diff
@@ -40,7 +42,9 @@ case class MainNetParams(
                           override val sidechainCreationVersion: SidechainCreationVersion = SidechainCreationVersion1,
                           override val chainId: Long = 33333333,
                           override val isCSWEnabled: Boolean = true,
-                          override val isNonCeasing: Boolean = false
+                          override val isNonCeasing: Boolean = false,
```

Comment:
In this particular case I don't like to much to add network settings for just one RPC call. Can we verify if is it needed from the product side?

Reply:
I checked with the product side and @paolocappelletti and we agreed to put these limits in config file, precisely in `EthServiceSettings`
I created two unit tests that check response size and timeout limitations